test/functions/diff.js

```diff
@@ -19,10 +19,19 @@ test('diff versions test', (t) => {
     ['1.1.0', '1.1.0-pre', 'minor'],
     ['1.1.0-pre-1', '1.1.0-pre-2', 'prerelease'],
     ['1.0.0', '1.0.0', null],
+    ['1.0.0-1', '1.0.0-1', null],
     ['0.0.2-1', '0.0.2', 'patch'],
+    ['0.0.2-1', '0.0.3', 'patch'],
```

Comment:
The difference is actually two .inc` patches, but that's still a `patch` difference.

```
> semver.inc('0.0.2-1', 'patch')
'0.0.2'
> semver.inc('0.0.2', 'patch')
'0.0.3'
```